test/microsoft/sharepoint/FindFilesOrFolders.test.js

```
@@ -0,0 +1,454 @@
+            }
+        };
+
+        await action.receive(context);
+
+        const data = context.sendJson.args[0][0];
+
+        assert.equal(context.sendJson.callCount, 1, 'should call sendJson once');
+        assert.equal(data.count, totalFiles, `should return ${totalFiles} files in total`);
+    });
+
+    it.only(`Finds all files in /${parentPaths.rootSubfolder1} recursively with name 'rootSubfolder2'`, async () => {
```

Biome (1.9.4)

[error] Don't focus the test.

The 'only' method is often used for debugging or during implementation. It should be removed before deploying to production.
Consider removing 'only' to ensure all tests are executed.
Unsafe fix: Remove focus from test.


(lint/suspicious/noFocusedTests)